main.py

```diff
@@ -128,6 +128,12 @@ def pyro_model(x, i, y):
 
 
 def localnews(INFERENCE):
+    device = torch.device('cpu')
+    torch.set_default_tensor_type(torch.DoubleTensor)
```

Comment:
All tensors defaulted to doubles run on CPU. (Do these actually need to be doubles, or would floats suffice?)